CMakeLists.txt

```diff
@@ -14,11 +14,13 @@ include_directories(.)
 add_subdirectory(switchapi)
 add_subdirectory(switchlink)
 add_subdirectory(switchsai)
+add_subdirectory(switch_utils)
```

Comment:
I'd suggest making this `switchutils` (no underscore), to be consistent with the other subdirectory names.

Reply:
Totally agree, will change it